Dockerfile

```diff
@@ -181,7 +190,9 @@ COPY --from=build /tmp/grpc_web_plugin /usr/local/bin/grpc_web_plugin
 
 COPY --from=build /tmp/protoc-gen-scala /usr/local/bin/
 
-COPY --from=build /go/pkg/mod/github.com/grpc-ecosystem/grpc-gateway/v2@${grpc_gateway_version}/protoc-gen-openapiv2/options /opt/include/protoc-gen-openapiv2/options/
+COPY --from=build /tmp/protoc-gen-clojure /usr/local/bin/
+
+COPY --from=build /go/pkg/mod/github.com/grpc-ecosystem/grpc-gateway/v2@v${grpc_gateway_version}/protoc-gen-openapiv2/options /opt/include/protoc-gen-openapiv2/options/
```

Comment:
I think `v2@v$` needs to be `v2@$` (remove the `v` prefix)? Seems to be failing looking up `vv...` as a version identifier.

Reply:
Whoops, that's a good catch - I've fixed it now and I think it's ready to trigger another build perhaps